test-app/polyfills.js

```diff
@@ -25,5 +25,8 @@ global.process = {
   umask() { return 0; }
 };
 
+// ant.design needs a buffer for Table -> rowSelection
+global.Buffer = global.Buffer || require('buffer').Buffer;
```

Comment:
Reminder for before merging:
- put this in the JIRA ticket with the list of things to set up on ELMU and OMA with the next educandu update.